templates/techreport/components/filter_meta.html

```diff
@@ -8,4 +8,10 @@
   <li>
     Rank: <span data-slot="rank">{{ request.args.get('rank', '')  or 'ALL' }}</span>
   </li>
+  {# if only one technology (including ALL), then legend is not shown so add label #}
+  {% if ',' not in request.args.get('tech', '') %}
```

Comment:
I would do this check around Client: as well (but hide label if its single technology)

otherwise you end up with `Client: Mobile`, even though it shows both mobile AND desktop when a single technology is selected